pmsoService/product_flow/features/import_request/api/serializers/read.py

```
@@ -0,0 +1,68 @@
+            "customer_name",
+            "receiver_name",
+            "import_method",
+            "note",
+            "created_at",
+            "updated_at",
+            "import_date",
+            "required_quantity",
+            "is_imported",
+            "sale_order",
+            "products",
+        ]
```

Ruff (0.13.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)